less/index.less

```diff
@@ -1 +1,74 @@
 // Start here!
+
+.container {
+    max-width: 980px;
+    width: 100%;
+}
+
+.nav-style{
+	display: flex;
+	justify-content: center;
+	margin-bottom:2%;
+	margin-top:2%;
+	
+	a{
+	color: gray;
+	}
+}
+
+.jum-style{
+	width: 100%; 
+	display: flex;
+    	flex-direction: column;
+    	align-items: center;
+	border-radius: 5px;
+
+	@media(max-width: 768px){
```

Comment:
when using bootstrap, you likely don't have to use media queries.    when you set col-sm col-md its baked in.